include/game.h

```diff
@@ -0,0 +1,27 @@
+#ifndef __GAME_H
```

Comment:
The idea of this is that the Game class will hold state data about the game in progress (how many players there are, the per-player data, the global game mode, etc).  Cleaner than having a butt-load of global variables